ProjectManager/ProjectManager/MainHome/ViewModel/Observable.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Observable.swift
+//  ProjectManager
+//
+//  Created by seohyeon park on 2022/09/17.
+//
+
+class Observable<T> {
+  typealias Listener = (T) -> Void
+  var listener: Listener?
+  var value: T {
+    didSet {
+      listener?(value)
+    }
+  }
+  init(_ value: T) {
+    self.value = value
+  }
+  func bind(listener: Listener?) {
+    self.listener = listener
+    listener?(value)
+  }
+}
```

Comment:
```suggestion
class Observable<T> {
  typealias Listener = (T) -> Void
  var listener: Listener?
  
  var value: T {
    didSet {
      listener?(value)
    }
  }
  
  init(_ value: T) {
    self.value = value
  }
  
  func bind(listener: Listener?) {
    self.listener = listener
    listener?(value)
  }
}
```
=> 함수 마다 한칸씩은 띄워주시는게 가독성면에서 좋습니다 :)